macosx/Makefile

```diff
@@ -70,20 +71,33 @@ LDFLAGS += $(UNIVERSAL_LDFLAGS)
 
 PYTHON = $(JHBUILD_PREFIX)/bin/python2.7
 OPENSSL_STATIC = $(JHBUILD_PREFIX)/lib/libssl.a $(JHBUILD_PREFIX)/lib/libcrypto.a
-
-$(IMAGE_NAME): path-check tool-checks $(IMAGE_STAGING_DIR)/$(PKG_NAME)
+SVN_STATIC = $(JHBUILD_PREFIX)/lib/libsvn_client-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_wc-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_diff-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_delta-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_subr-1.a \
+$(JHBUILD_PREFIX)/lib/libexpat.a \
+$(JHBUILD_PREFIX)/lib/libsvn_ra-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_ra_svn-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_ra_local-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_fs-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_repos-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_fs_util-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_fs_fs-1.a \
+$(JHBUILD_PREFIX)/lib/libsvn_fs_x-1.a \
+$(JHBUILD_PREFIX)/lib/libintl.a
+APR_STATIC = $(JHBUILD_PREFIX)/lib/libapr-1.a $(JHBUILD_PREFIX)/lib/libaprutil-1.a $(JHBUILD_PREFIX)/lib/libapriconv-1.a
+
+$(IMAGE_NAME): tool-checks $(IMAGE_STAGING_DIR)/$(PKG_NAME)
 	rm -f $@
 	hdiutil create -imagekey zlib-level=9 -srcfolder $(IMAGE_STAGING_DIR) $@
 
-path-check:
-	if (echo $(PATH) | grep -F "$(MACPORTS_PREFIX)"); then false; else echo "Path is OK"; fi
-
 tool-checks: have-$(CC) have-jhbuild have-gtk-mac-bundler
 
 have-%:
 	which $*
 
-$(IMAGE_STAGING_DIR)/$(PKG_NAME): check-nmap check-ncat check-ndiff check-zenmap check-nping check-nmap-update COPYING.formatted
+$(IMAGE_STAGING_DIR)/$(PKG_NAME): check-nmap-update COPYING.formatted # check-nmap check-ncat check-ndiff check-zenmap check-nping 
```

Comment:
Why are the rest of these commented out? Guessing that you wanted to test more quickly, but they should be restored in the final merge.